modular_skyrat/modules/clock_cult/code/structures/traps/trap.dm

```diff
@@ -13,7 +13,24 @@
 	if(!IS_CLOCK(user))
 		return
 
-	for(var/obj/structure/destructible/clockwork/trap/trap in get_turf(src)) // No 50-spear instakills please
+	if(user.loc != get_turf(src))
```

Comment:
```suggestion
	if(get_turf(user) != get_turf(src))
```

Reply:
This was intended so that people don't lay traps while in a locker or w/e